hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/HL7Message.java

```diff
@@ -9,6 +9,8 @@
 	private String sendingApplication;
 
 	private String sendingFacility;
+	
+	private String sendingFacilityConf;
```

Comment:
I'm not too sure what the different is between the two sendingFacility properties. I assume one is the incoming one on the message and the other is used to create outgoing message (e.g. for an error scenario). Is there any reason we wouldn't just modify the HL7Message at this point as once we hit the error scenario it effectively represents the outgoing message?